srcpkgs/rssguard/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'rssguard'
 pkgname=rssguard
-version=4.2.2
+version=4.2.3
 revision=1
 build_style=cmake
 configure_args="-DUSE_WEBENGINE="false""
```

Comment:
could you fix quoting here? just `...="-DUSE_WEBENGINE=false"` could work

Reply:
Thanx, done!